### PR TITLE
cookie implemented for navbar

### LINKED ISSUE #5295: Inappropriate Display of 'Get Started' Button for Users with Layer5io Account
### Description

The [site](https://layer5.io/) currently displays a signup button regardless of whether the user has a cookie indicating prior signup. This behavior is not consistent with user expectations and may lead to confusion.

### Expected Behavior

After a user has signed up and a relevant cookie is present, the 'Get Started' button should not be displayed to enhance a streamlined and personalized user experience.

### Possible Solution:
Implement logic to check for the presence of a signup-related cookie. If the cookie is present, do not display the signup button. Ensure this check is performed consistently across page loads.

### Screenshots

(This is even i have logged in Layer5io)

image

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Description**

This PR fixes  #5295 

I added just a simple logic to the button or not.
we will be making it into a profile button if the session cookie address changes to the main domain i.e. layer5.io rather than meshery.layer5.io(which is a subdomain) then we might so logout button and other default options.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/General/Navigation/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { IoMdClose } from "@react-icons/all-files/io/IoMdClose";
 import Button from "../../../reusecore/Button";
 import { Container } from "../../../reusecore/Layout";
 import { useStyledDarkMode } from "../../../theme/app/useStyledDarkMode";
-
+import Cookie from "js-cookie";
```
Comment (captain-Akshay) [inline src/sections/General/Navigation/index.js:9]: this was added here.

File: src/sections/General/Navigation/index.js [hunk 2]
```diff
@@ -282,21 +333,46 @@ const Navigation = () => {
...
+              secondary
+              className="banner-btn two"
+              external={true}
+              title="Get Started"
+              alt="Signup for Layer5 Cloud"
+              url="https://meshery.layer5.io/registration"
+            />
+          ) : (
+            <></>
+          )}
```
Comment (captain-Akshay) [inline src/sections/General/Navigation/index.js:351]: this was added here.

File: src/sections/General/Navigation/index.js [hunk 3]
```diff
@@ -18,150 +18,159 @@ import { ReactComponent as Logo } from "../../../assets/images/app/layer5-colorM
...
+    ],
   };
   data["Solutions"] = {
-    nodes: []
+    nodes: [],
   };
   const [expand, setExpand] = useState(false);
   const [scroll, setScroll] = useState(false);
-
+  const [cookie, setCookie] = useState(null);
```
Comment (captain-Akshay) [inline src/sections/General/Navigation/index.js:173]: this was added here.

**Comment by l5io** [comment]
🚀 Preview for commit 18790362a56d379fbf4df23c76d6132d7cfd5166 at: https://65b0c2db4d190a55d2951223--layer5.netlify.app

**Review by leecalcote — COMMENTED** [review]
Here's what I see while having an active session - 

Screenshot 2024-01-25 at 3 19 05 PM

Two items:
1. The active session has access to the capability "meshmap", however, the URL on the playground button is still play.meshery.io.
2. Should we be expecting to see an avatar of the user at some point?

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
@captain-Akshay  this doesn't seems to work?

**Comment by leecalcote** [comment]
@captain-Akshay there's a piling up of your open, incomplete PRs across a couple repos...

**Comment by l5io** [comment]
🚀 Preview for commit cd31ce73d721b773e0ecbba694c3b652a2f1cb70 at: https://65dc3c934e315f7ff0af1620--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 950941e175fd29981a8173fcbd05e4eafc83cc12 at: https://65dc84c74e315fba6daf1979--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 00b3c36ae7cbf05320ff74e85c89a8f2eb9dd2da at: https://65dcac79bdab100ae3813153--layer5.netlify.app

**Comment by captain-Akshay** [comment]
[Screencast from 2024-02-26 20-34-33.webm](https://github.com/layer5io/layer5/assets/59491379/94ba56a6-2d87-44a7-a294-2b69d3cf0415)

[Screencast from 2024-02-26 20-39-07.webm](https://github.com/layer5io/layer5/assets/59491379/89a8830a-75fc-4bef-bff1-6937aa2b7907)

@ritiksaxena124

**Comment by l5io** [comment]
🚀 Preview for commit cc9508dfe0c46cb7acbd417710c54fa9c6f4225a at: https://65ef1e5cea89e068425eb83c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4bdddaf56eab4942daeef49ce3d3b4ff3d7c410c at: https://65f1942a3e64390ee4b71e09--layer5.netlify.app

**Comment by captain-Akshay** [comment]
[Screencast from 2024-03-13 17-02-13.webm](https://github.com/layer5io/layer5/assets/59491379/70be2e7f-1d87-410c-b422-b5318ff98293)

**Comment by l5io** [comment]
🚀 Preview for commit 74c5d434e50eeae7124a1d96aefc4d6f49ea1653 at: https://65f1b706f269a22b1a288ce2--layer5.netlify.app

**Pull request merged**